pom.xml

```diff
@@ -2,7 +2,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.spotify</groupId>
-  <version>2.6.3-SNAPSHOT</version>
+  <version>3.0.0-SNAPSHOT</version>
```

Comment:
`Futures.transformAsync` appeared only in Guava 19, stated compatibility of current version is Guava 18.